src/Compiler/CompilerUtils.cpp

```diff
@@ -35,7 +34,11 @@
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/TargetSelect.h"
 #include "llvm/Support/ToolOutputFile.h"
+#include "llvm/Support/raw_ostream.h"
 #include "llvm/Target/TargetMachine.h"
+#include <fstream>
+#include <memory>
+#include <regex>
```

Comment:
Could you include these three lines first? They are from C++, and we often include them first before mlir stuffs.